### PR TITLE
Adding initial event info for 2020-09-29

### PR DESCRIPTION
**Pull request by jcohen02**
This PR includes the event information for the event scheduled for 2020-09-29.

The event is awaiting an ID and Indico registration link and there will likely be a couple of other updates to be made but in terms of making the basic event information available, this is ready to be merged once an ID and registration link are added.

### CODE REVIEW

File: _events/talks/event-ID_UNKNOWN.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+authors:
+    - &speaker
+      name:  James Smithies
+      is_speaker: true
+      affiliation: 1
+    - name:  Arianna Ciula
+      is_speaker: true
+      affiliation: 1
+affiliations:
+    - name: King’s Digital Laboratory, King's College London, London, UK
```
Comment (Chilipp) [inline _events/talks/event-ID_UNKNOWN.md:12]:
```suggestion
    - name: King's Digital Laboratory, King's College London, London, UK
```

File: _events/talks/event-ID_UNKNOWN.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+registration_url: https://indico.scc.kit.edu/event/ID_UNKNOWN/
+---
+This SORSE event describes King’s Digital Lab (KDL), a Research Software Engineering lab operating within the Faculty of Arts and Humanities at King’s College London (UK). The KDL team of 18 project managers, analysts, designers, engineers, and systems managers specialise in arts & humanities, cultural heritage, and creative industries research and development. The talk will provide a current state overview of the lab, and describe our RSE HR roles (see [https://zenodo.org/record/2564790](https://zenodo.org/record/2564790)) and a relatively recent trial initiative that defines the different ways the team can contribute to research. A more technically-oriented overview of our ongoing work in refining the fundamental components of  KDL Software Development Lifecycle (see https://github.com/kingsdigitallab/sdlc-for-rse/) will also be provided, including descriptions of our modelling workflows, processes, and efforts to increase sustainability, reproducibility, and research data management best practice ([https://zenodo.org/record/3361580#.X09x6mdKhhE](https://zenodo.org/record/3361580#.X09x6mdKhhE)).
```
Comment (Chilipp) [inline _events/talks/event-ID_UNKNOWN.md:24]:
```suggestion
This SORSE event describes King's Digital Lab (KDL), a Research Software Engineering lab operating within the Faculty of Arts and Humanities at King's College London (UK). The KDL team of 18 project managers, analysts, designers, engineers, and systems managers specialise in arts & humanities, cultural heritage, and creative industries research and development. The talk will provide a current state overview of the lab, and describe our RSE HR roles (see [https://zenodo.org/record/2564790](https://zenodo.org/record/2564790)) and a relatively recent trial initiative that defines the different ways the team can contribute to research. A more technically-oriented overview of our ongoing work in refining the fundamental components of  KDL Software Development Lifecycle (see https://github.com/kingsdigitallab/sdlc-for-rse/) will also be provided, including descriptions of our modelling workflows, processes, and efforts to increase sustainability, reproducibility, and research data management best practice ([https://zenodo.org/record/3361580#.X09x6mdKhhE](https://zenodo.org/record/3361580#.X09x6mdKhhE)).
```

File: _events/talks/event-ID_UNKNOWN.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+### About the speakers
+
+**James Smithies** is Director of King’s Digital Lab & Reader and Deputy Director of eResearch at King’s College London. He was previously Senior Lecturer in Digital Humanities and Associate Director of the UC CEISMIC Digital Archive at the University of Canterbury, New Zealand. He has worked in the government and commercial IT sectors as a technical writer and editor, business analyst, and project manager. In 2017 he published a monograph for Palgrave Macmillan titled [Digital Humanities and the Digital Modern](https://www.palgrave.com/gp/book/9781137499431). James is interested in applied and theoretical digital humanities, including data analysis, minimal computing, digital archives, modelling and virtual worlds. He has subsidiary interests in critical and media theory, and postphenomenology. His historical work focuses on the history of ideas, technology (including but not limited to computing), and literature. See his [personal website](https://jamessmithies.org/) for more information about his work.
```
Comment (Chilipp) [inline _events/talks/event-ID_UNKNOWN.md:28]:
```suggestion
**James Smithies** is Director of King's Digital Lab & Reader and Deputy Director of eResearch at King’s College London. He was previously Senior Lecturer in Digital Humanities and Associate Director of the UC CEISMIC Digital Archive at the University of Canterbury, New Zealand. He has worked in the government and commercial IT sectors as a technical writer and editor, business analyst, and project manager. In 2017 he published a monograph for Palgrave Macmillan titled [Digital Humanities and the Digital Modern](https://www.palgrave.com/gp/book/9781137499431). James is interested in applied and theoretical digital humanities, including data analysis, minimal computing, digital archives, modelling and virtual worlds. He has subsidiary interests in critical and media theory, and postphenomenology. His historical work focuses on the history of ideas, technology (including but not limited to computing), and literature. See his [personal website](https://jamessmithies.org/) for more information about his work.
```

File: _events/talks/event-ID_UNKNOWN.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+**Arianna Ciula** is Deputy Director & Senior Research Software Analyst at King’s Digital Lab. She has broad experience in digital humanities research and teaching, research management, and digital research infrastructures. She holds a PhD in Manuscript and Book Studies (digital palaeography, University of Siena), an MA in Applied Computing in the Humanities (King’s College London) and a BA Hons in Communication sciences (computational linguistics, University of Siena). She worked at King’s in the past as Research Associate (Centre for Computing in the Humanities, 2003-2009). From 2009 to 2012, she worked as Science officer at the European Science Foundation (Humanities) and from 2013 she worked as Research Facilitator at the University of Roehampton for three years..
```
Comment (Chilipp) [inline _events/talks/event-ID_UNKNOWN.md:30]:
```suggestion
**Arianna Ciula** is Deputy Director & Senior Research Software Analyst at King's Digital Lab. She has broad experience in digital humanities research and teaching, research management, and digital research infrastructures. She holds a PhD in Manuscript and Book Studies (digital palaeography, University of Siena), an MA in Applied Computing in the Humanities (King's College London) and a BA Hons in Communication sciences (computational linguistics, University of Siena). She worked at King's in the past as Research Associate (Centre for Computing in the Humanities, 2003-2009). From 2009 to 2012, she worked as Science officer at the European Science Foundation (Humanities) and from 2013 she worked as Research Facilitator at the University of Roehampton for three years..
```

File: _events/talks/event-ID_UNKNOWN.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+Her personal research interests focus on the modelling of scholarly digital resources related to primary sources. She lectured and published on humanities computing, in particular on digital manuscript studies and editing; she has organised conferences and workshops in digital humanities, and is an active member of its international community. She was co-PI in the project ‘[Modelling between Digital and Humanities: Thinking in Practice](http://modellingdh.eu/)’ funded by the Volkswagen Foundation (2016-18). See list of [publications](https://kclpure.kcl.ac.uk/portal/en/persons/arianna-ciula(8b156ef5-c75d-46a7-9be9-2eb2a4666f31)/publications.html).
```
Comment (Chilipp) [inline _events/talks/event-ID_UNKNOWN.md:31]:
```suggestion
Her personal research interests focus on the modelling of scholarly digital resources related to primary sources. She lectured and published on humanities computing, in particular on digital manuscript studies and editing; she has organised conferences and workshops in digital humanities, and is an active member of its international community. She was co-PI in the project "[Modelling between Digital and Humanities: Thinking in Practice](http://modellingdh.eu/)" funded by the Volkswagen Foundation (2016-18). See list of [publications](https://kclpure.kcl.ac.uk/portal/en/persons/arianna-ciula(8b156ef5-c75d-46a7-9be9-2eb2a4666f31)/publications.html).
```

**Review by Chilipp — CHANGES_REQUESTED** [review]
hey @jcohen02! thanks a lot for your work here. The failures are totally not your fault. It's a python 2.7 thing, it cannot decode characters such as `’`. So I replaced them here.

 @vsoch: Do you think we can adapt the pdf-generation action to use python 3.7 (or 3.8) instead of 2.7? Then this would not be a problem anymore.

File: _events/talks/event-ID_UNKNOWN.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Digital Humanities RSE: King’s Digital Lab as experiment and lifecycle"
```
Comment (Chilipp) [inline _events/talks/event-ID_UNKNOWN.md:2]:
```suggestion
title: "Digital Humanities RSE: King's Digital Lab as experiment and lifecycle"
```

File: _events/talks/event-ID_UNKNOWN.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+### About the speakers
+
+**James Smithies** is Director of King's Digital Lab & Reader and Deputy Director of eResearch at King’s College London. He was previously Senior Lecturer in Digital Humanities and Associate Director of the UC CEISMIC Digital Archive at the University of Canterbury, New Zealand. He has worked in the government and commercial IT sectors as a technical writer and editor, business analyst, and project manager. In 2017 he published a monograph for Palgrave Macmillan titled [Digital Humanities and the Digital Modern](https://www.palgrave.com/gp/book/9781137499431). James is interested in applied and theoretical digital humanities, including data analysis, minimal computing, digital archives, modelling and virtual worlds. He has subsidiary interests in critical and media theory, and postphenomenology. His historical work focuses on the history of ideas, technology (including but not limited to computing), and literature. See his [personal website](https://jamessmithies.org/) for more information about his work.
```
Comment (Chilipp) [inline _events/talks/event-ID_UNKNOWN.md:28]:
```suggestion
**James Smithies** is Director of King's Digital Lab & Reader and Deputy Director of eResearch at King's College London. He was previously Senior Lecturer in Digital Humanities and Associate Director of the UC CEISMIC Digital Archive at the University of Canterbury, New Zealand. He has worked in the government and commercial IT sectors as a technical writer and editor, business analyst, and project manager. In 2017 he published a monograph for Palgrave Macmillan titled [Digital Humanities and the Digital Modern](https://www.palgrave.com/gp/book/9781137499431). James is interested in applied and theoretical digital humanities, including data analysis, minimal computing, digital archives, modelling and virtual worlds. He has subsidiary interests in critical and media theory, and postphenomenology. His historical work focuses on the history of ideas, technology (including but not limited to computing), and literature. See his [personal website](https://jamessmithies.org/) for more information about his work.
```

**Review by Chilipp — APPROVED** [review]

**Comment by Chilipp** [comment]
@eileen-kuehn: Could you please provide a registration url and an event-id?

- [ ] registration url inserted
- [ ] event-ID changed

**Comment by vsoch** [comment]
I'm not sure what action you are asking for on the pdf-generator - it installs python3 and uses pandoc to generate the output, and @jcohen02 reports that all is working - if you are able to find the issue [in the entrypoint](https://github.com/rseng/pdf-generator/blob/master/entrypoint.sh) please let me know and I'd be happy to create a branch for testing.

**Comment by Chilipp** [comment]
> it seemed to generate a PDF without complaining about the quote characters.

yep @jcohen02, this is probably because you used the default settings which does not extract any metadata from the markdown file. If you want to mimick the PDF generation locally, you need to run the docker build as such:

```bash
cp _events/talks/event-ID_UNKNOWN.md ci/
docker run --env-file ci/env.txt -it -v `pwd`:/github/workspace rseng/pdf-generator
``` 

This should fail with the same error as in the github action (well, now it works, as I changed the problematic characters)

**Comment by Chilipp** [comment]
> if you are able to find the issue in the entrypoint please let me know and I'd be happy to create a branch for testing.

see https://github.com/rseng/pdf-generator/issues/6, is it okay that way @vsoch?

**Comment by Chilipp** [comment]
@jcohen02, the PDF generation is failing because it's a forked PR. We'll do this later (actually it should not attempt to open a PR for forks, sorry about that)

File: _events/talks/event-ID_UNKNOWN.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+      index: 1
+time:
+    - start: 2020-09-29T15:00:00Z
+      end: 2020-09-29T16:30:00Z
+author: *speaker
+category: talks
+language: English
+date: 2020-09-08
+last_modified_at: 2020-09-08
+registration_url: https://indico.scc.kit.edu/event/ID_UNKNOWN/
```
Comment (eileen-kuehn) [inline _events/talks/event-ID_UNKNOWN.md:22]:
```suggestion
registration_url: https://indico.scc.kit.edu/event/923/
```

File: _events/talks/event-ID_UNKNOWN.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+authors:
+    - &speaker
+      name:  James Smithies
+      is_speaker: true
+      affiliation: 1
+    - name:  Arianna Ciula
+      is_speaker: true
+      affiliation: 1
+affiliations:
+    - name: King's Digital Laboratory, King's College London, London, UK
```
Comment (eileen-kuehn) [inline _events/talks/event-ID_UNKNOWN.md:12]:
```suggestion
    - name: King's Digital Laboratory, King's College London, UK
```

**Review by eileen-kuehn — COMMENTED** [review]
I added the correct indico URL for registration and did open registration already.
I still need some information about the actual schedule that I can properly fill the timetable.

The registered ID for the event is *27* so the file name should become `event-027.md`. @jcohen02, could you change it?

**Comment by Chilipp** [comment]
> The registered ID for the event is 27 so the file name should become event-027.md. @jcohen02, could you change it?

I am not @jcohen02, but I changed it :wink:

**Comment by jcohen02** [comment]
Thanks for your work investigating the issues here @Chilipp, and for changing the event ID 🙂

**Comment by Chilipp** [comment]
alright, when the CI finishes, we can merge this PR. Thanks a lot @jcohen02 (although for your patience :sweat_smile:)

**Comment by jcohen02** [comment]
No problem @Chilipp, not quite as straightforward as we had all hoped but a useful exercise nonetheless. Thanks for your help in resolving the issues.

**Comment by Chilipp** [comment]
there seems to be a problem with the site of personal website of @tscheypidi (https://www.pik-potsdam.de/research/members/dietrich) which is why the link checking in the CI is failing. But I suspect, PIK will resolve this soon, so I'll merge this PR despite the failure (see also #327)

**Comment by PYTHON3610** [comment]
I'm new to python and programming where and what can I do here

**Comment by vsoch** [comment]
hey @PYTHON3610 ! I think this event is now passed so we don't have further work on this particular site (until it or a similar event happens again) but if you want to join the society's slack (instructions here) https://society-rse.org/join-us/ you can probably get notified when there is a new/future effort to make something.

**Comment by PYTHON3610** [comment]
Thanks you for the response and I will join your societies slack

**Pull request merged**